app/controllers/documents_controller.rb

```diff
@@ -38,7 +38,11 @@ def new
   end
 
   def create
-    @document = @planning_application.documents.build(tags: document_params[:tags], file: document_params[:file])
+    @document = @planning_application.documents.build(tags: document_params[:tags], file: document_params[:file],
+                                                      numbers: document_params[:numbers],
+                                                      publishable: document_params[:publishable],
+                                                      referenced_in_decision_notice:
+                                                          document_params[:referenced_in_decision_notice])
```

Comment:
`params` are a bit of a mess in this controller.

We have direct setting of params here, `documents_list_params`, `document_params`, `document_upload_params` and `document_form_params`! Surely we have simplified this controller now to the point we can just have one `document_params` and use it for `create` and `update`? Unless I am missing something.

I realise this is totally outside the scope of this ticket, but seems like a good opportunity to tidy it up?

Reply:
I missed `form_params`!

Reply:
I also think we have unused actions (I think all the confirm actions are not used any more?), and `set_document` is being used inconsistently (edit and update should all be using `set_document`, right?)

Again, all outside the ticket, but would be nice :-)

Reply:
I'm really pleased we were able to do this - it's much simpler :)